plugin/evm/config.go

```diff
@@ -24,8 +25,11 @@ const (
 	defaultMaxBlocksPerRequest                  = 0 // Default to no maximum on the number of blocks per getLogs request
 	defaultContinuousProfilerFrequency          = 15 * time.Minute
 	defaultContinuousProfilerMaxFiles           = 5
-	defaultTxRegossipFrequency                  = 1 * time.Minute
-	defaultTxRegossipMaxSize                    = 15
+	defaultTxRegossipFrequency                  = 1 * time.Second
+	defaultTxRegossipMaxSize                    = 16
+	defaultPriorityTxRegossipFrequency          = 1 * time.Second
+	defaultPriorityTxRegossipSize               = 32
+	defaultPriorityTxRegossipAddressTxs         = 16
```

Comment:
I know these are already long, but should we make these names more specific ie. TxsPerGossipMsg, and TxsPerAddress

Reply:
Same comment for the related names below